src/stylesheets/components/current-task/currentTask.scss

```diff
@@ -0,0 +1,34 @@
+.current-task-container {
+	width: 100%;
+	height: 100%;
+}
+
+.current-task-title {
+	width: 100%;
+	text-align: left;
+	font-size: 1.5rem;
+}
+
+.current-task-content-container {
+	width: 100%;
+	height: calc(100% - 48px);
```

Comment:
What's the 48px based on? If it's just to have some spacing then doing something like `5vh` might be better

Reply:
Its based on the padding

Reply:
ah right, should be fine for now then